app/views/allocations/_allocated.html.erb

```diff
@@ -20,14 +17,14 @@
     </tr>
     </thead>
     <tbody class="govuk-table__body">
-    <% @prisoners.each_with_index  do |prisoner, i| %>
+    <% @offenders_allocated.each_with_index do |offender, i| %>
       <tr class="govuk-table__row offender_row_<%= i %>">
```

Comment:
Worth extracting to partial?

Reply:
Think this TR is different in each table isn't it?